java/rocksjni/rocksjni.cc

```diff
@@ -1289,6 +1290,52 @@ void Java_org_rocksdb_RocksDB_deleteRange__J_3BII_3BII(
                               jend_key, jend_key_off, jend_key_len);
 }
 
+
+/*
+ * Class:     org_rocksdb_RocksDB
+ * Method:    deleteFilesInRange
+ * Signature: (J[BII[BII)V
+ */
+JNIEXPORT void JNICALL Java_org_rocksdb_RocksDB_deleteFilesInRange(
+  JNIEnv* env, jobject jdb, jlong jdb_handle, jbyteArray jbegin_key,
+  jint jbegin_key_off, jint jbegin_key_len, jbyteArray jend_key,
+  jint jend_key_off, jint jend_key_len) {
+  auto* db = reinterpret_cast<rocksdb::DB*>(jdb_handle);
+  auto* cf_handle = db->DefaultColumnFamily();
+
+  jbyte* begin_key = new jbyte[jbegin_key_len];
```

Comment:
would just `jbyte begin_key[jbegin_key_len]`; work here to keep it on the stack? Same goes for `jend_key`.

Reply:
👍 good catch. will do